src/reducers/entities/channels.js

```diff
@@ -306,11 +306,8 @@ function myMembers(state = {}, action) {
     }
     case ChannelTypes.POST_UNREAD_SUCCESS: {
         const data = action.data;
-        const channelState = state[data.channelId];
+        const channelState = state[data.channelId] || {};
```

Comment:
This isn't going to cause any change in behaviour is it? I'm a bit worried this might mean that we're going to end up with a partial channel member object that's missing things like roles or channel-specific settings.

Reply:
Yes, I would prefer to check for null instead of having it return an empty object

Reply:
0/5 on this, so I'm changing it